### PR TITLE
Updated readme to look more like other projects

### PR DESCRIPTION
**Pull request by ghost**
Only thing I didn't have time for was to write up a comprehensive example of button implementation. Just did a very simple one. Everything else should be there once the project is up on NPM, etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,40 @@
-# react-native-button
+# Native Button
+## A native button for React.
```
Comment (maluramichael) [inline README.md]: 'for React Native'

File: README.md [hunk 2]
```diff
@@ -1,3 +1,40 @@
-# react-native-button
+# Native Button
```
Comment (maluramichael) [inline README.md]: Easy Button

File: README.md [hunk 3]
```diff
@@ -1,3 +1,40 @@
...
+## A native button for React.
+[![Gitter](https://badges.gitter.im/Devnetik/react-native-button.svg)](https://gitter.im/Devnetik/react-native-button?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```
Comment (maluramichael) [inline README.md]: Like i said in the chat just rename every react-native-button to react-native-easy-button and this pr is good.

File: README.md [hunk 4]
```diff
@@ -1,3 +1,40 @@
...
 
-[![Gitter](https://badges.gitter.im/Devnetik/react-native-button.svg)](https://gitter.im/Devnetik/react-native-button?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
\ No newline at end of file
+[![npm](https://img.shields.io/npm/dm/react-native-button.svg?maxAge=2592000)](https://www.npmjs.com/package/react-native-button)
+[![npm](https://img.shields.io/npm/v/react-native-button.svg?maxAge=2592000)](https://www.npmjs.com/package/react-native-button)
+[![Beerpay](https://beerpay.io/Devnetik/react-native-button/badge.svg?style=flat)](https://beerpay.io/Devnetik/react-native-button)
+
+## Getting started
+```
+sh
```
Comment (maluramichael) [inline README.md]: Move the sh one line up

**Comment by ghost** [comment]
@maluramichael added commits for the changes you requested. Should be good to go. Let me know if there's anything else that needs done.

**Pull request merged**